### PR TITLE
Miscellaneous changes (September-30-2015)

### PR DESCRIPTION
**Pull request by opadron**
This is the first among a series of PR's that I'd like to apply in sequence.

Applies a number of various fixes, house-keeping tasks, and other omission-correcting changes.

### CODE REVIEW

File: filter_plugins/zookeeper.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+def zk_url(host_list, host_vars, net_interface, client_port):
```
Comment (cpatrick) [inline filter_plugins/zookeeper.py:2]: sphinx-style docs go here. :smile:

File: playbooks/gobig/assign-groups.yml [hunk 2]
```diff
@@ -2,32 +2,40 @@
...
 
         master_set: "{{ groups.get(mesos_masters, []) }}"
         slave_set: "{{ groups.get(mesos_slaves, []) }}"
-        mesos_set: "{{ master_set | union(slave_set) }}"
 
-        spark_set: "{{ master_set }}"
+        mesos_set: "{{ master_set | union(slave_set) }}"
 
+        spark_set: "{{ groups.get(spark, []) | union(mesos_set) }}"
+        uvcmetrics_set: "{{ groups.get(uvcmetrics, []) }}"
```
Comment (cpatrick) [inline playbooks/gobig/assign-groups.yml:24]: There are enough of these that I want to alphabetize.

**Comment by cpatrick** [comment]
A couple of nits, but lgtm.

File: playbooks/gobig/assign-groups.yml (hunk 2 above)
Comment (opadron) [inline playbooks/gobig/assign-groups.yml:24]: I'm going to have to disagree, here, since some of these sets are dependent on others.

Comment (cpatrick) [inline playbooks/gobig/assign-groups.yml:24]: ah. gotcha.

**Comment by opadron** [comment]
@cpatrick Could you make a quick pass through the doc strings?

**Comment by cpatrick** [comment]
lgtm.

**Pull request merged**